### DISCUSSION TITLE
Additional Archive Utility Filetype Icons

### DISCUSSION THREAD

**Discussion by AClownfa** [elrumo/macOS_Big_Sur_icons_replacements#1522] [Icon request] [answered]
Would like additional Archive Utility style filetype icons to use along with those created by [cph101](https://macosicons.com/#/u/cph101) for RAR and BROTLI archives to modernize filetypes used by The Unarchiver.

![rar](https://github.com/user-attachments/assets/d26da2b8-4683-4edc-9d07-294f9bf937f7)

Need ACE, ADF, ADZ, AMIGA, ARCHIVE, CAB, DMS, JAR, LHA, RPM and SIT.

**Comment by vikasbhatia** [comment, marked as answer]
Here you go. ICNS files will be available [on my profile](https://macosicons.com/#/u/sidewinder) once approved.

![ACE](https://github.com/user-attachments/assets/de01c13b-8833-44fe-823d-0c6ac3f8821e)
![ADF](https://github.com/user-attachments/assets/c75cd165-a536-4b85-aaeb-8bb356e20d8c)
![ADZ](https://github.com/user-attachments/assets/6095cbf0-8979-43b9-b0ea-6a0f175940d3)
![AMIGA](https://github.com/user-attachments/assets/2f372618-2200-4077-8e99-797288b550e9)
![Archive](https://github.com/user-attachments/assets/8d4b6112-470a-4585-8905-1bf405756ec8)
![CAB](https://github.com/user-attachments/assets/7c12e5d9-ef31-4b2c-9cd4-3758bafe8825)
![DMS](https://github.com/user-attachments/assets/f5caf50b-d5b4-4810-b08b-9a3b3150194e)
![JAR](https://github.com/user-attachments/assets/0d4c5f2e-fd7e-4af1-b508-db7cfdf45acc)
![LHA](https://github.com/user-attachments/assets/52d70a0c-df77-4aca-9090-35711708660d)
![RAR](https://github.com/user-attachments/assets/aa220ad4-6324-47ec-84e8-37f41fae2288)
![RPM](https://github.com/user-attachments/assets/341ad6e4-1e12-445e-8253-7167d440b51c)
![SIT](https://github.com/user-attachments/assets/baccea4f-b108-45b0-812c-88d335da1a26)

**Comment by AClownfa** [comment]
Perfect! Thanks so much, [vikasbhatia](https://github.com/vikasbhatia).
